change/@fluentui-react-tooltip-0c7f8278-85ad-44f1-82b3-8048cf63b2e7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 7 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28264&commitId=2d472677278c9ab931640351b6dd2e9897f31234&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1939 screenshots unchanged, 0 screenshots with different dimensions and 7 screenshots with visible difference.


  <details><summary><b>unknown</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|3| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|22| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|231| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|29| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|27| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|21| Changed|
|Tooltip Converged.withArrow.chromium.png|3| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>